### ISSUE TITLE
unavailable deep speech 9  clinet 

### ISSUE THREAD

**Issue by doctorwho-123** [mozilla/DeepSpeech-examples#119]
I downloaded the example of Deep Speech 9, but I wanted to run its .NET file, I realized that there are two projects, only one project called Deep Speech Client is unavailable? Please tell me what should I do? Where can I find this project?

[
![image](https://user-images.githubusercontent.com/20268490/109035313-c337dc00-76dd-11eb-9671-f76d8735cd31.png)
](url)

**Comment by lissyx** [comment]
Did you installed the NuGet?

**Comment by doctorwho-123** [comment]
@lissyx  yes

**Comment by lissyx** [comment]
and you properly setup your project? then I have no idea, I don't know C#. Maybe @carlfm01 knows?

**Comment by doctorwho-123** [comment]
@lissyx  yes . i do not know where to find this Deep speech clinet project?

**Comment by lissyx** [comment]
it's part of the nuget as much as I can tell ...

**Comment by lissyx** [comment]
`DeepSpeechWPF` example is what we use in CI to test ... https://github.com/mozilla/DeepSpeech/blob/7b2eeb6734a49c3d9439644e9fe0685cb46ad695/taskcluster/tc-build-utils.sh#L286-L303

**Comment by doctorwho-123** [comment]
@lissyx thank you. But then why is one of the two projects inactive and I can not run this program? unavailable deep speech 9 clinet.

**Comment by lissyx** [comment]
> @lissyx thank you. But then why is one of the two projects inactive and I can not run this program? unavailable deep speech 9 clinet.

no project is inactive, I don't get what you are referring to. I've shared as much as I can, this code runs, I showed you the code that performs the setup. As I said I know nothing about C#, I don't use Windows, I can't help more.

**Comment by doctorwho-123** [comment]
@lissyx  The project file could not be loaded. Could not find a part of the path . Please upload the project to  github again

**Comment by lissyx** [comment]
> @lissyx The project file could not be loaded. Could not find a part of the path . Please upload the project to github again

I'm not going to repeat myself for ever: this code works, this is what we use in CI, and I showed you how it's being setup. There is nothing to re-upload.

**Comment by doctorwho-123** [comment]
This is the error I get. He does not know your client at all.

![image](https://user-images.githubusercontent.com/20268490/109046312-d6e93f80-76e9-11eb-907b-e556014bf6a7.png)

**Comment by lissyx** [comment]
> This is the error I get. He does not know your client at all.
> 
> ![image](https://user-images.githubusercontent.com/20268490/109046312-d6e93f80-76e9-11eb-907b-e556014bf6a7.png)

you keep asking but you have never confirmed whether you properly applied the suggestion from the link I shared: `nuget install DeepSpeechWPF/packages.config -OutputDirectory DeepSpeechWPF/packages/`

What do you don't understand in "I don't know C# and I can't help more" ?

BTW sharing errors as screenshots makes it super painful for people to help you, please share as text only.

**Comment by reuben** [comment]
I think I did something wrong when bumping the examples folder to r0.9, probably a botched merge from master. The reference there should be to NuGet but it's a local one to be used in CI.

**Comment by doctorwho-123** [comment]
@lissyx  thanks. i understood .
I checked the link you sent and applied it, but unfortunately I did not succeed, it does not work for me and

**Comment by doctorwho-123** [comment]
@reuben  Thank you. What should I do now?

**Comment by reuben** [comment]
I've updated the examples repo, hopefully with a fix: https://github.com/mozilla/DeepSpeech-examples/commit/0f1168d78f7bd4c1b1984614dba663e7ce685380

Can you update your copy of DeepSpeechWPF and try again?

**Comment by reuben** [comment]
Also, when filing issues in the future, please describe exactly the steps you followed so that we don't have to infer it indirectly from error reports.

**Comment by doctorwho-123** [comment]
@reuben thanks. the first time I try to open the project, it shows me this error. The next time I click OK, I see that the Deep Speech client is disabled.


![image](https://user-images.githubusercontent.com/20268490/109049119-1feec300-76ed-11eb-86af-9e174dd44f02.png)


![image](https://user-images.githubusercontent.com/20268490/109049457-9095df80-76ed-11eb-8a2b-1e60b504949d.png)

**Comment by doctorwho-123** [comment]
@reuben  yes. Hi, I copied and run again after you updated, but unfortunately it did not work.

**Comment by reuben** [comment]
Then I'm afraid you'll have to figure it out on your end or give more details. Other people are able to build and run the project, so it could be some Visual Studio problem. Document here exactly what you've tried, what errors you got, and maybe someone with more .NET experience will be able to help. And please stop posting screenshots as already requested. Copy and paste the text here, properly formatted.

**Comment by reuben** [comment]
For example, the error message explicitly tells you to look at the Output window, yet you didn't paste its contents here on the issue. If you don't provide all the information you have nobody will be able to help you.

**Comment by carlfm01** [comment]
Hello @doctorwho-123 that is just a warning, you should be able to run the project with Reuben's change, here's a PR that removes the warning: 
https://github.com/mozilla/DeepSpeech-examples/pull/122

**Comment by shubhmak** [comment]
NuGet package is installed, but have you imported it??
